lib/ts/ink_align.h

```diff
@@ -53,6 +53,12 @@ union Alias64 {
 /** Default alignment */
 #define INK_ALIGN_DEFAULT(size) INK_ALIGN(size, INK_MIN_ALIGN)
 
+static inline size_t
+aligned_spacing(size_t len, size_t block = INK_MIN_ALIGN)
+{
+  return INK_ALIGN(len, block);
+}
```

Comment:
Nevertheless, I'm not certain we need two functions that do the exact same thing, even if the existing thing's name isn't quite as descriptive as one might prefer.